.gitattributes

```diff
@@ -0,0 +1,16 @@
+/.github                export-ignore
+/tests                  export-ignore
+/.editorconfig          export-ignore
+/.env.testing           export-ignore
+/.gitattributes         export-ignore
+/.gitignore             export-ignore
+/.wp-env.json           export-ignore
+/codeception.dist.yml   export-ignore
+/composer.json          export-ignore
+/composer.lock          export-ignore
```

Comment:
Does this cause problems with Composer if we're not including the composer file?

Reply:
See also: [bradp/dotenv](https://github.com/bradp/dotenv/blob/e3c608deb5cb8bebc81ee96e4ad41cbd579f4334/.gitattributes#L8-L9)!!